tooling-cpu-load/src/main/java/org/jboss/eap/qe/microprofile/tooling/cpu/load/utils/JavaProcessBuilder.java

```diff
@@ -0,0 +1,98 @@
+package org.jboss.eap.qe.microprofile.tooling.cpu.load.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Helper builder class for starting Java process.
+ */
+public class JavaProcessBuilder {
+
+    private String mainClass;
+    private String workingDirectory;
+    private Map<String, String> systemProperties = new HashMap<String, String>();
+    private List<String> classpathEntries = new ArrayList<String>();
+    private List<String> mainClassArguments = new ArrayList<String>();
+    private String javaRuntime = "java";
```

Comment:
If this is supposed to be a constant please name the variable to look like it (capitals) and make it final.

Reply:
I'll make configurable as I've removed the setter method accidentally.

Reply:
fixed